emt4j-common/src/main/java/org/eclipse/emt4j/common/SourceInformation.java

```diff
@@ -30,6 +30,9 @@ public class SourceInformation implements Serializable {
 
     private String[] extras;
 
+    // TODO should use identifier
+    private String fullIdentifier;
```

Comment:
`fullIdentifier` was temporarily added during internal development. I suggest that we should no longer use it.

Reply:
removed fullIdentifier